Problem with Windows environment

A new Windows user followed the getting started directions in [this document](http://taxcalc.readthedocs.io/en/latest/contributor_guide.html), but could not pass all the unit tests because the mock package had not been installed.  Here is the the error message:
```
================================== FAILURES ===================================
_____________________________ test_force_no_numba _____________________________

    def test_force_no_numba():
        """
        Force execution of code for non-existence of Numba
        """
        global Magic_calc6

        # Mock the numba module
>       from mock import Mock
E       ImportError: No module named mock

taxcalc\tests\test_decorators.py:296: ImportError
```
@PeterDSteinberg, Can you explain why this happened on Windows?  Would the same thing happen to a new Mac users or a new Linux user?  What files control the addition of packages needed by Tax-Calculator but not in the standard Anaconda Python distribution?

--- a/conda.recipe/meta.yaml
+++ b/conda.recipe/meta.yaml
@@ -7,6 +7,7 @@ requirements:
     - pytest
     - setuptools
     - python
+    - mock
     - numpy ==1.10.4
     - pandas <=0.18.0
     - numba
@@ -16,6 +17,7 @@ requirements:
     - pytest
     - setuptools
     - python
+    - mock
     - numpy ==1.10.4
     - pandas <=0.18.0
     - numba

--- a/environment.yml
+++ b/environment.yml
@@ -13,5 +13,6 @@ dependencies:
 - mock
 - pep8
 - pylint
+- mock
 - pip:
   - pytest-pep8

--- a/environment.yml
+++ b/environment.yml
@@ -10,7 +10,6 @@ dependencies:
 - six
 - ipython
 - ipython-notebook
-- mock
 - pep8
 - pylint
 - mock

